benchmark/benchmain/main.go

```diff
@@ -47,6 +47,7 @@ import (
 	"fmt"
 	"io"
 	"log"
+	math_rand "math/rand"
```

Comment:
Please just leave this import name as is.

Reply:
done